index.html

```diff
@@ -215,10 +215,10 @@ <h1 class="skills-header"><span><img src="./images/Rectangle-three.svg" alt=""><
         </div>
         <div class="form-container">
             <form action="https://formspree.io/f/xnqlwgep" method="POST">
-                <input type="text" name="name" placeholder="Full name">
-                <input type="email" name="email" placeholder="Email">
-                <textarea name="message" id="text-msg" cols="500" rows="5" placeholder="Enter text here"></textarea>
-                <input type="button" value="Get in touch" id="btn-submit">
+                <input type="text" name="name" placeholder="Full name" required>
+                <input type="email" name="email" placeholder="Email" required>
+                <textarea name="message" id="text-msg" cols="500" rows="5" placeholder="Enter text here" required></textarea>
```

Comment:
- [ ] Name input should have a maxlength="30" attribute as  stated in the requirements.
- [ ] Message input should have a maxlength="500" attribute as stated in the requirements.